internal/tools/clickhouse_create_blocks_table.sql

```diff
@@ -25,5 +26,5 @@ CREATE TABLE base.blocks (
     INDEX idx_timestamp timestamp TYPE minmax GRANULARITY 1,
     INDEX idx_number number TYPE minmax GRANULARITY 1,
 ) ENGINE = ReplacingMergeTree(insert_timestamp, is_deleted)
-ORDER BY (chain_id, hash) PRIMARY KEY (chain_id, hash)
+ORDER BY (chain_id, hash)
```

Comment:
why?

Reply:
We do not need a primary key, we have the sort order and indexes defined